app.js

```diff
@@ -216,6 +216,57 @@ if (params) {
   }
 }
 
+
+var isDay = true;
+
+$("toggle").addEventListener("click", function(){
+	if (isDay) {
```

Comment:
Please match the indentation of the rest of the file, i.e. two spaces, no tabulations.